packages/aws-cdk-lib/aws-eks/lib/alb-controller.ts

```diff
@@ -35,52 +35,57 @@ export class AlbControllerVersion {
   /**
    * v2.1.1
    */
-  public static readonly V2_1_1 = new AlbControllerVersion('v2.1.1', '1.4.1', false);
+  public static readonly V2_1_1 = new AlbControllerVersion('v2.1.1', '1.1.2', false);
 
   /**
    * v2.1.2
    */
-  public static readonly V2_1_2 = new AlbControllerVersion('v2.1.2', '1.4.1', false);
+  public static readonly V2_1_2 = new AlbControllerVersion('v2.1.2', '1.1.4', false);
 
   /**
    * v2.1.3
    */
-  public static readonly V2_1_3 = new AlbControllerVersion('v2.1.3', '1.4.1', false);
+  public static readonly V2_1_3 = new AlbControllerVersion('v2.1.3', '1.1.5', false);
 
   /**
    * v2.0.0
    */
-  public static readonly V2_2_0 = new AlbControllerVersion('v2.2.0', '1.4.1', false);
+  public static readonly V2_2_0 = new AlbControllerVersion('v2.2.0', '1.2.0', false);
 
   /**
    * v2.2.1
    */
-  public static readonly V2_2_1 = new AlbControllerVersion('v2.2.1', '1.4.1', false);
+  public static readonly V2_2_1 = new AlbControllerVersion('v2.2.1', '1.2.3', false);
 
   /**
    * v2.2.2
    */
-  public static readonly V2_2_2 = new AlbControllerVersion('v2.2.2', '1.4.1', false);
+  public static readonly V2_2_2 = new AlbControllerVersion('v2.2.2', '1.2.5', false);
 
   /**
    * v2.2.3
    */
-  public static readonly V2_2_3 = new AlbControllerVersion('v2.2.3', '1.4.1', false);
+  public static readonly V2_2_3 = new AlbControllerVersion('v2.2.3', '1.2.6', false);
 
   /**
    * v2.2.4
    */
-  public static readonly V2_2_4 = new AlbControllerVersion('v2.2.4', '1.4.1', false);
+  public static readonly V2_2_4 = new AlbControllerVersion('v2.2.4', '1.2.7', false);
 
   /**
    * v2.3.0
    */
-  public static readonly V2_3_0 = new AlbControllerVersion('v2.3.0', '1.4.1', false);
+  public static readonly V2_3_0 = new AlbControllerVersion('v2.3.0', '1.3.0', false);
```

Comment:
Not all chart version would aligned with App version, for the version `v2.4.0` the chart version should be `1.3.2`.

You may check version mapping with the command below
```sh
helm repo update
helm search repo aws-load-balancer-controller --versions
```

Reply:
Fixed 👍